accounts-db/src/ancient_append_vecs.rs

```diff
@@ -753,6 +758,12 @@ impl AccountsDb {
             self.addref_accounts_failed_to_shrink_ancient(vec![accounts_to_combine.remove(i)]);
         });
         target_slots_sorted.sort_unstable();
+        self.shrink_ancient_stats
+            .slots_cannot_move_count
+            .fetch_add(slots_cannot_move_count, Ordering::Relaxed);
```

Comment:
We could use `accounts_keep_slots.len()` instead.
```
        self.shrink_ancient_stats
            .slots_cannot_move_count
            .fetch_add(accounts_keep_slots.len(), Ordering::Relaxed);
```

Reply:
yep, looks like we could have. wanna open a new pr?

Reply:
sure. will do ...